relatedpermissions.php

```diff
@@ -329,7 +329,14 @@ function relatedpermissions_civicrm_pre($op, $entity, $objectID, &$entityArray)
   }
 
   if ($op == 'edit' || $op == 'create') {
-    $entityArray['is_permission_a_b_v'] = CRM_Utils_Array::value('is_permission_' . $_GET['rtype'] . '_v', $_POST, 0);
-    $entityArray['is_permission_b_a_v'] = CRM_Utils_Array::value('is_permission_' . strrev($_GET['rtype']) . '_v', $_POST, 0);
+    $rtype = array();
+    foreach($_POST as $k => $v) {
```

Comment:
I think we shouldn't access these global variables like $_GET, $_POST, etc directly, unless you're absolutely sure we need the raw value. There's a class provided  by CiviCRM core that acts as a wrapper that allows access to request values.

Eg:
```php

foreach (CRM_Utils_Request::exportValues() as $k => $v) {
  ...
}

 $entityArray['is_permission_a_b_v'] = CRM_Utils_Request::retrieveValue('is_permission_' . $rtype . '_v', 'Boolean', 0);
```